packages/vector_graphics_compiler/lib/src/svg/masking_optimizer.dart

```diff
@@ -21,22 +21,18 @@ class _Result {
 /// Converts a vector_graphics PathFillType to a path_ops FillType.
 path_ops.FillType toPathOpsFillTyle(PathFillType fill) {
   switch (fill) {
-    case PathFillType.evenOdd:
-      return path_ops.FillType.evenOdd;
+    case PathFillType.evenOdd: return path_ops.FillType.evenOdd;
```

Comment:
FYI I believe this dartfmt change was reverted

Reply:
Yeah, hasn't rolled into flutter master as of this commit.